tests/workflows/stream_retry_node_annotation/tests/test_workflow.py

```diff
@@ -0,0 +1,148 @@
+from vellum.workflows.events.types import CodeResourceDefinition, VellumCodeResourceDefinition
+from vellum.workflows.nodes.utils import ADORNMENT_MODULE_NAME
+from vellum.workflows.utils.uuids import uuid4_from_hash
+from vellum.workflows.workflows.event_filters import all_workflow_event_filter
+
+from tests.workflows.stream_retry_node_annotation.workflow import InnerNode, Inputs, StreamingRetryExample
+
+
+def test_workflow_stream__happy_path():
```

Comment:
a copy paste of `stream_try_node_annotation` with the difference of not emitting streaming events on the retry node itself